README.md

```diff
@@ -9,28 +9,31 @@ Minimalist CLI email client, written in Rust.
 
 ## Table of contents
 
-* [Motivation](#motivation)
-* [Installation](#installation)
-* [Configuration](#configuration)
-* [Usage](#usage)
-  * [List mailboxes](#list-mailboxes)
-  * [List messages](#list-messages)
-  * [Search messages](#search-messages)
-  * [Download attachments](#download-attachments)
-  * [Read a message](#read-a-message)
-  * [Write a new message](#write-a-new-message)
-  * [Reply to a message](#reply-to-a-message)
-  * [Forward a message](#forward-a-message)
-  * [Copy a message](#copy-a-message)
-  * [Move a message](#move-a-message)
-  * [Delete a message](#delete-a-message)
-  * [Listen to new messages](#listen-to-new-messages)
-* [Interfaces](#interfaces)
-  * [GUI](#gui)
-  * [TUI](#tui)
-* [License](https://github.com/soywod/himalaya/blob/master/LICENSE)
-* [Changelog](https://github.com/soywod/himalaya/blob/master/CHANGELOG.md)
-* [Credits](#credits)
+- [📫 Himalaya ![gh-actions](https://github.com/soywod/himalaya/actions?query=workflow%3Adeployment)](#-himalaya-)
+  - [Table of contents](#table-of-contents)
+  - [Motivation](#motivation)
+  - [Installation](#installation)
+  - [Nix](#nix)
+    - [Installing using Nix](#installing-using-nix)
+    - [Development using Nix](#development-using-nix)
+  - [Configuration](#configuration)
+  - [Usage](#usage)
+    - [List mailboxes](#list-mailboxes)
+    - [List messages](#list-messages)
+    - [Search messages](#search-messages)
+    - [Download attachments](#download-attachments)
+    - [Read a message](#read-a-message)
+    - [Write a new message](#write-a-new-message)
+    - [Reply to a message](#reply-to-a-message)
+    - [Forward a message](#forward-a-message)
+    - [Copy a message](#copy-a-message)
+    - [Move a message](#move-a-message)
+    - [Delete a message](#delete-a-message)
+    - [Listen to new messages](#listen-to-new-messages)
+  - [Interfaces](#interfaces)
+    - [GUI](#gui)
+    - [TUI](#tui)
+  - [Credits](#credits)
```

Comment:
Huh, not sure what happened here. I didn't realize VSCode magically modified the TOC from behind the scenes.

Reply:
I had a VSCode extension that did this.